src/app/tests.py

```diff
@@ -1543,3 +1549,25 @@ def test_no_license_id_on_license_requests_edit_xml(self):
         self.assertEqual(resp.status_code,200)
         self.assertIn("app/license_namespace_requests.html",(i.name for i in resp.templates))
         self.assertEqual(resp.resolver_match.func.__name__,"licenseNamespaceRequests")
+
+class TestCronJob(TestCase):
+    def test_delete_old_files(self):
+        """Check if the files older than 10 days are getting deleted or not"""
+        # create a test directory with some files
+        test_dir = os.path.join(os.getcwd() + '/app/media', 'AnonymousUser')
+        os.makedirs(test_dir, exist_ok=True)
+        for i in range(1, 11):
+            file_path = os.path.join(test_dir, f'test_file_{i}.txt')
+            with open(file_path, 'w') as f:
+                f.write('test')
+            # set file creation time to 11 days ago
+            creation_time = datetime.datetime.now() - datetime.timedelta(days=11)
```

Comment:
Let's keep some files newer than 10 days as well and assert for them.

Reply:
Now I have updated the unit tests, like 5 files are older than 10 days and others are not and they test is passing as well

![image](https://user-images.githubusercontent.com/65075935/227793213-f721b69c-0304-47e9-b309-b505846ff4b8.png)